src/Analyzer.JsonRuleEngine.UnitTests/JsonRuleEngineTests.cs

```diff
@@ -277,57 +274,44 @@ private void AssertEvaluationsAndResultsAreAsExpected(IEvaluation evaluation, in
                 }
             }]", DisplayName = "Invalid regex pattern")]
         [ExpectedException(typeof(JsonRuleEngineException))]
-        public void EvaluateRules_InvalidRule_ExceptionIsThrown(string invalidRule)
+        public void Create_InvalidRules_ExceptionIsThrown(string invalidRule)
         {
-            // Arrange
-            var template =
-                JObject.Parse(
-                @"{
-                    ""$schema"": ""https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#"",
-                    ""resources"": [
-                        {
-                            ""type"": ""Microsoft.ResourceProvider/resource0"",
-                            ""properties"": {
-                                ""somePath"": ""someValue"",
-                                ""someOtherPath"": ""someOtherValue""
-                            }
-                        }
-                    ]
-                }");
-
-            string expectedFileId = "MyTemplate";
-
-            TemplateContext templateContext = new TemplateContext
-            {
-                OriginalTemplate = template,
-                ExpandedTemplate = template,
-                IsMainTemplate = true,
-                TemplateIdentifier = expectedFileId
-            };
+            // Act
+            JsonRuleEngine.Create(invalidRule, t => null);
+        }
 
-            // Setup mock line number resolver
-            var mockLineResolver = new Mock<ILineNumberResolver>();
+        [ExpectedException(typeof(JsonRuleEngineException))]
+        public void Create_NullRules_ExceptionIsThrown()
+        {
```

Comment:
These tests are missing the TestMethod tag so they're not actually being run in the test suite.

Reply:
😳 🤦‍♂️ Oh my, I can't believe I missed that...

Thanks for catching it.